CHANGELOG.md

```diff
@@ -75,6 +76,17 @@
 - [0.2.0](#020)
 - [0.1.0](#010)
 
+## Unreleased
```

Comment:
If the changes and the changelog details look fine, I will update the version and release date in the PR itself.

Reply:
I believe the changes look good. When @czeslavo's comment is covered I believe we can merge 👍